client/public/config/configuration.json

```diff
@@ -2,6 +2,7 @@
   {
     "URL":"https://namex-dev.pathfinder.gov.bc.ca",
     "PAYMENT_PORTAL_URL": "https://dev.bcregistry.ca/business/auth/makepayment",
+    "VUE_APP_PAY_ROOT_API": "https://pay-api-dev.pathfinder.gov.bc.ca/api/v1",
```

Comment:
You'll need to configure this in OS Dev and Test. (There is no config map for namerequest in Prod yet.)
And add check items in the release ticket (5732).

Reply:
Definitely, tho Patrick said he had/going to take care of it, as this was his and Saravan's request. I broke the rules here and snuck it in 😁

Reply:
Release ticket updated, will verify OS configs 👍